src/styles/index.scss

```diff
@@ -1,3 +1,266 @@
+  position: relative;
+  line-height: 14.06px;
+}
+
+.nav__item-button {
+  text-align: center;
+  color: #060b35;
+  cursor: pointer;
+  height: 100%;
+  line-height: 60px;
+  text-decoration: none;
+  display: inline-block;
+  position: relative;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__item-button:hover,
+.nav__item .is-active {
+  color: var(--bg-color);
+}
+
+.is-active::after {
+  content: '';
+  background-color: var(--bg-color);
+  opacity: 1;
+  border-radius: 8px;
+  width: 100%;
+  height: 4px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+.main {
+  justify-content: center;
+  height: 100vh;
+  display: flex;
+}
+
+.main__container {
+  grid-template-columns: repeat(var(--count-size-monitor), 200px);
+  gap: 46px 48px;
+  width: fit-content;
+  height: fit-content;
+  margin: 50px 40px;
+  display: grid;
+}
+
+@media (min-width: 488px) {
+  .main__container {
+    --count-size-monitor: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .main__container {
+    --count-size-monitor: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .main__container {
+    --count-size-monitor: 4;
+  }
+}
+
+.card {
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  transition: all 0.3s;
+}
+
+.card__image {
+  background-repeat: no-repeat;
+  background-size: auto;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+  display: block;
+}
+
+.card__reviews {
+  color: #060b35;
+  height: 16px;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card__txt {
+  color: #616070;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+}
+
+.card__price {
+  color: #060b35;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+}
+
+.card__description {
+  color: #060b35;
+  width: 166px;
+  height: 36px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.card__description--margin--top {
+  margin-top: 40px;
+}
+
+.card__code {
+  color: #616070;
+  width: 98px;
+  height: 14px;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card__code--margin--top {
+  margin-top: 4px;
+}
+
+.card__rating {
+  justify-content: space-between;
+  align-items: center;
+  width: 166px;
+  height: 16px;
+  display: flex;
+}
+
+.card__rating--margin--top {
+  margin-top: 16px;
+}
+
+.card__priceBlock {
+  justify-content: space-between;
+  align-items: center;
+  width: 166px;
+  height: 18px;
+  display: flex;
+}
+
+.card__priceBlock--margin--top {
+  margin-top: 24px;
+}
+
+.card__button {
+  color: #fff;
+  background-color: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  justify-content: center;
+  align-items: center;
+  width: 166px;
+  height: 40px;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  text-decoration: none;
+  display: flex;
+}
+
+.card__button--margin--top {
+  margin-top: 16px;
+}
+
+.card:hover {
+  transform: scale(1.2);
+  transition: 0.3s;
```

Comment:
The hover effect on `.card` uses `transform: scale(1.2)`, which can cause layout shifts and affect the user experience. Consider using a different visual effect that doesn't alter the element's size, such as changing the background color or adding a shadow.